UE/Application/States/BaseState.cpp

```diff
@@ -35,13 +35,18 @@ void BaseState::handleAttachReject()
     logger.logError("Uexpected: handleAttachReject");
 }
 
-void BaseState::handleSmsReceive(uint8_t action, std::string text) {
-    logger.logError("Uexpected: handleSmsReceive");
-}
-
 void BaseState::handleDisconnected()
 {
     logger.logError("Uexpected: handleDisconnected");
 }
 
+void BaseState::handleSmsReceive(uint8_t action, const std::string& text, common::PhoneNumber fromPhoneNumber, common::PhoneNumber toPhoneNumber) {
+    logger.logError("Uexpected: handleSmsReceive");
```

Comment:
wszedzie w logach sa literowki 'Uexpected' zamiast 'UNexpected'

Reply:
to nowy ficzer